setup.py

```diff
@@ -59,6 +59,6 @@ def read(fname):
     install_requires=['sagemaker-containers==2.1', 'retrying==1.3.3'],
```

Comment:
https://github.com/aws/sagemaker-pytorch-container/blob/master/setup.py#L38

Your unit tests might fail?

Reply:
updated sagemaker-containers version (I assume that's what you're referring to)